docker-compose.yml

```diff
@@ -61,11 +34,10 @@ slave2:
     - MESOS_ISOLATOR=cgroups/cpu,cgroups/mem
     - MESOS_LOG_DIR=/var/log
     - MESOS_RESOURCES=ports(*):[31000-31100]
-    - MESOS_MODULES=file:///isolator/build/isolator/modules.json
+    - MESOS_MODULES=file:///calico/modules.json
     - MESOS_ISOLATION=com_mesosphere_mesos_NetworkIsolator
     - MESOS_HOOKS=com_mesosphere_mesos_NetworkHook
     - ETCD_AUTHORITY=etcd:4001
-    - MESOS_ATTRIBUTES=public:true
```

Comment:
We tagged one as public in order to know deterministically which slave that task will land on since we created a route to that slave from outside the VM (via `add-container-route.sh`).  It looks like someone has since deleted that constraint from `stars.json` and `stars-isolated.json`.  Now it's up to chance that it continues to be scheduled on slave1.  I am in favor of re-introducing the attribute and the constraint to reduce the magic (relying on available resources and Marathon's current offer matching logic).